code/hud/hudets.cpp

```diff
@@ -133,11 +133,16 @@ void update_ets(object* objp, float fl_frametime)
 		engine_aggregate_strength = ship_get_subsystem_strength(ship_p, SUBSYSTEM_ENGINE);
 	}
 
+	// very annoying, but ship_get_subsystem_strength will typically cap at no lower than 15% strength reported
+	// causing the condition below to possibly erroneously believe the engine strength isn't changing while being
+	// repaired below that threshold. use the ACTUAL strength for this check
+	float actual_engine_strength = ship_get_subsystem_strength(ship_p, SUBSYSTEM_ENGINE, false, true);
```

Comment:
This should be inside the `else` block a few lines up - it should follow the rules of the `Maneuver_despite_engines` flag.  If the flag is active, it should be 1.0f, just like its counterpart.

Optional: call the variable `actual_engine_aggregate_strength` and rename the other variable, perhaps `effective_engine_aggregate_strength`

Reply:
>.> these names are too long
We don't need 'aggregate', that can be assumed.